src/ui-components/dashboard/dashboard.css

```diff
@@ -0,0 +1,33 @@
+.container {
+  margin: 21px;
```

Comment:
convert to rem

Reply:
My understanding is that for margins, you want to use pixels because this will be consistent across devices. If you use REM, you can end up with very small margins which is not desired in this case.